packages/xml-views-validation/test/validators/attributes/unknown-attribute-key-spec.ts

```diff
@@ -0,0 +1,444 @@
+import { expect } from "chai";
+import { find } from "lodash";
+import { DocumentCstNode, parse } from "@xml-tools/parser";
+import { buildAst } from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import {
+  generateModel,
+  expectExists,
+} from "@ui5-language-assistant/test-utils";
+import {
+  computeExpectedRange,
+  testValidationsScenario,
+} from "../../test-utils";
+import { validateUnknownAttributeKey } from "../../../src/validators/attributes/unknown-attribute-key";
+
+describe("the unknown attribute name validation", () => {
+  let ui5SemanticModel: UI5SemanticModel;
+
+  before(async () => {
+    ui5SemanticModel = await generateModel({ version: "1.74.0" });
+  });
+
+  context("true positive scenarios", () => {
+    function assertSingleIssue(xmlSnippet: string, message: string): void {
```

Comment:
we will want to re-use these assertion utils in other specs as well (future upgrade)
Maybe the stack traces will be less useful on errors, but ours tests are getting verbose otherwise...